test_review.py

```diff
@@ -0,0 +1,2 @@
+print("hello dev!")
+print("please review my change
```

Comment:
It seems you have typos in your 2nd line. Kindly check the print syntax on line 2

Reply:
Well yeah you are right